playbooks/bootstrap-python.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: all
+  become: true
+  gather_facts: False
+  tasks:
+    # Fortunately, python is a dependency of yum. So RedHat systems always have it.
+    - name: install python
+      raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
+      register: output
+      changed_when: output.stdout != ""
+
+    - debug:
+        msg: Now you shouldn't need to use the 'raw' module for these hosts anymore.
```

Comment:
Not entirely sure we need this ;)

Reply:
Haha true. I got a little excited about my debug messages :)